apps/examples/memory_fragmentation_test/memory_fragmentation_test.c

```diff
@@ -0,0 +1,344 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file memory_fragmentation_test.c
+
+/// @brief Intentionally allocate/free small and large memory segments in a mixed-up manner.
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <tinyara/config.h>
```

Comment:
Because of header priority (or hierarchy), we had better include <tinyara/config.h> at very first.
Usually the sequence is
1. <tinyara/config.h>
2. standard headers like stdio.h, unistd.h
3. tinyara specific headers like <tinyara/xxx.h>
4. internal header like "xxx.h"

Reply:
Thanks for your comments. It's done.